### DISCUSSION TITLE
Problem with permalinks

### DISCUSSION THREAD

**Discussion by YodelingCowboys** [code4recovery/12-step-meeting-list#507] [Q&A] [answered]
This has been a circus.  I added meetings to TSML as "individual meeting" and also added a district number and published the meeting.  The district disappeared.  Tried to change to "part of a group" and added the district again and updated.  It reverted to "individual meeting with no district.  So I exported to .csv, edit that file to add the district numbers, and imported.  Again, the meetings were all "individual meeting" with no district number.

Next I deleted TSML, reinstalled, added the districts, and again imported the .csv file.  No change -- all individual meetings without district numbers.  No idea how but I ended up in the permalinks settings.  It was a custom setting and I failed to write it down.  I googled how to reset and everything I looked at said to set it to plain and save.  I did and had to play around with the menu to get the meeting list to display.  Had to turn off caching and clear my browser cache.  The link for the meetings page changed from https://lincaa.org/meetings to https://lincaa.org/?post_type=tsml_meeting.  When I click on a meeting name, instead of https://lincaa.org/meetings/<meeting-slug>, it is now https://lincaa.org/meetings/<meeting-slug>/?post_type=tsml_meeting.  Hence, I get 404 errors.

What can I do to get the permalinks corrected?

Thanks,
Charlie

**Comment by YodelingCowboys** [comment]
Correction to #507
When I click on a meeting name, instead of https://lincaa.org/meetings/meeting-slug, it is now https://lincaa.org/meetings/meeting-slug//?post_type=tsml_meeting. Hence, I get 404 errors.

**Comment by joshreisner** [comment]
This is what the permalinks page on our demo site looks like: 
<img width="1269" alt="Screen Shot 2021-08-24 at 5 05 41 PM" src="https://user-images.githubusercontent.com/1551689/130705493-1e1ff6b5-f3e4-433a-97d5-25906f2a4652.png">

We have `/%postname%` as the value there, sounds like you did too.

Also sounds like you might have edited the WordPress Address (URL) setting on Settings > General as well? (We just have `https://demo.code4recovery.org` for our site there.)

**Comment by YodelingCowboys** [comment]
No, the Wordpress Address is https://lincaa.org.

If I change the permalinks to be Custom Structure as https://lincaa.org/%postname%, will it correct the problems I am having with TSML?  Or should I leave set as it is (plain) and export, remove TSML, reinstall TSML, and import the meetings again?  Which is more likely to not cause any problems?  Please note all other menu options work as before -- only TSML has had problems.

Thanks.

**Comment by YodelingCowboys** [comment, marked as answer]
I took the easy way out.  I left the permalinks alone.  I deleted TSML, reinstalled TSML, imported my meetings, and now everything works as expected.  So far.  Newbie errors but slowly I learn.
